peachjam/templates/peachjam/_citations_list.html

```diff
@@ -0,0 +1,27 @@
+{% load peachjam i18n %}
+{% for item in citations %}
+  {{ item.doc_type }} <span class="badge bg-secondary">{{ item.docs|length }}</span>
+  <ol>
+    {% for doc in item.docs|slice:":10" %}
+      <li>
+        <a href="{% url 'document_detail' frbr_uri=doc.work.frbr_uri|strip_first_character %}">{{ doc.title }}</a>
+      </li>
+    {% endfor %}
+    {% if item.docs|length > 10 %}
+      {% for doc in item.docs|slice:"10:" %}
+        <li class="collapse {{ item.doc_type }}">
```

Comment:
If `{{ item.doc_type }}` is "Concept note", then the class will be `class="collapse Concept note"` which may cause it to have odd styling.

Instead, use the outermost for-loop's counter, which is unique for the group:

```class="collapse grp-{{ forloop.parentloop.counter }}"```

Reply:
You'll also need to pass in something to the template so that the class  can differentiate between the same groups in the incoming and outgoing citation lists.